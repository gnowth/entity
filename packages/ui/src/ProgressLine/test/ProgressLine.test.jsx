import 'jest-styled-components';
import renderer from 'react-test-renderer';
import React from 'react';

import UIProgressLine from '..';

it('renders correctly', () => {
  const tree = renderer.create(
    <UIProgressLine />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
