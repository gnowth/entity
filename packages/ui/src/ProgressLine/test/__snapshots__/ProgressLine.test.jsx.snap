// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  height: 4px;
  min-width: 2rem;
  overflow: hidden;
  position: relative;
  -webkit-transition: opacity 0.25s cubic-bezier(0.4,0,0.6,1) 0ms;
  transition: opacity 0.25s cubic-bezier(0.4,0,0.6,1) 0ms;
  -webkit-transform: translateZ(0);
  -ms-transform: translateZ(0);
  transform: translateZ(0);
}

.c1 {
  background-color: black;
  height: 100%;
  position: absolute;
  -webkit-transition: -webkit-transform 0.25ms cubic-bezier(0.4,0,0.6,1) 0ms;
  -webkit-transition: transform 0.25ms cubic-bezier(0.4,0,0.6,1) 0ms;
  transition: transform 0.25ms cubic-bezier(0.4,0,0.6,1) 0ms;
  -webkit-transform-origin: top left;
  -ms-transform-origin: top left;
  transform-origin: top left;
  width: 100%;
  -webkit-transition: -webkit-transform 225ms;
  -webkit-transition: transform 225ms;
  transition: transform 225ms;
  -webkit-transform: translateX(-0%);
  -ms-transform: translateX(-0%);
  transform: translateX(-0%);
}

.c2 {
  height: 100%;
  position: absolute;
  -webkit-transform: scaleX(0);
  -ms-transform: scaleX(0);
  transform: scaleX(0);
  -webkit-transform-origin: top left;
  -ms-transform-origin: top left;
  transform-origin: top left;
  width: 100%;
  -webkit-animation: jSYKOo 2s infinite linear;
  animation: jSYKOo 2s infinite linear;
  left: -145.166611%;
  -webkit-transition: none;
  transition: none;
}

.c3 {
  background-color: black;
  display: inline-block;
  height: 100%;
  position: absolute;
  width: 100%;
  -webkit-animation: hvVyfP 2s infinite linear;
  animation: hvVyfP 2s infinite linear;
}

.c4 {
  -webkit-animation: hgPsqz 2s infinite linear;
  animation: hgPsqz 2s infinite linear;
  height: 100%;
  left: -54.888891%;
  position: absolute;
  -webkit-transition: none;
  transition: none;
  -webkit-transform-origin: top left;
  -ms-transform-origin: top left;
  transform-origin: top left;
  visibility: visible;
  width: 100%;
}

.c5 {
  -webkit-animation: dkIHsc 2s infinite linear;
  animation: dkIHsc 2s infinite linear;
  background-color: black;
  display: inline-block;
  height: 100%;
  position: absolute;
  visibility: visible;
  width: 100%;
}

<div
  className="c0"
  height="4px"
>
  <div
    className="c1"
    value={100}
  />
  <div
    className="c2"
    value={null}
  >
    <span
      className="c3"
      value={null}
    />
  </div>
  <div
    className="c4"
  >
    <span
      className="c5"
    />
  </div>
</div>
`;
